packages/aserto/src/aserto/client/directory/__init__.py

```diff
@@ -1,14 +1,29 @@
-from typing import List, Literal, Tuple
+from typing import List, Literal, Tuple, Any, Optional
 
-from grpc import ChannelCredentials, ssl_channel_credentials
+from grpc import ChannelCredentials, ssl_channel_credentials, secure_channel, Channel
+import grpc.aio as grpc_aio
 
 
 class NotFoundError(Exception):
     pass
 
+class ConfigError(Exception):
+    pass
 
 Header = Literal["authorization", "aserto-tenant-id", "if-match", "if-none-match"]
 
+def validate_addresses(
+    address: str,
+    reader_address: str,
+    writer_address: str,
+    importer_address: str,
+    exporter_address: str,
+    model_address: str) -> Any:
+    if address == "" and reader_address == "" and writer_address == "" and importer_address == "" and exporter_address == "" and model_address == "":
+        raise ValueError("at least one directory service address must be specified")
+        
+    return None
```

Comment:
You can remove the return statement. Not returning anything is the same as returning None.